Anytype/Sources/PresentationLayer/Modules/Chat/Subviews/Collection/ChatCollectionViewCoordinator.swift

```diff
@@ -86,6 +86,7 @@ final class ChatCollectionViewCoordinator<
         
         let oldContentSize = collectionView.contentSize
         let oldContentOffset = collectionView.contentOffset
+        let olsIsNearTop = (collectionView.contentOffset.y - collectionView.topOffset.y) < 30
```

Comment:
Will it work differently on devices with/without dynamic isle?

Reply:
yep